### PR TITLE
Split tweet to sentences before parsing

### PR DESCRIPTION
**Pull request by kleinay**
before passing the tweet text to props_wrapper, we first segment it to sentences using Spacy.
Then, we merge the props_wrapper outputs to a single parse, by:
1. modifying indices to match the full tweet text
2. modifying the symbols (entity-id and predicate-id) to a unified list, sorted by the (first) word index.

### CODE REVIEW

File: src/baseline_automatic_pipeline_system/parse_okr_info.py [hunk 1]
```diff
@@ -61,12 +61,10 @@ def parse_tweets(raw_sentences):
...
     for sent_id, sent in normed_sentences.iteritems():
-        # try:
-        #     parsed_sentences[sent_id] = parse_tweet(sent)
-        # except Exception as e:
-        #     logging.error("failed to parse sentence: " + sent +" | Error: " + str(e))
-        parsed_sentences[sent_id] = parse_tweet(sent)
-
+        try:
+            parsed_sentences[sent_id] = parse_tweet(sent)
+	except Exception as e:
```
Comment (OriShapira) [inline src/baseline_automatic_pipeline_system/parse_okr_info.py]: spacing wrong

File: src/baseline_automatic_pipeline_system/parse_okr_info.py [hunk 2]
```diff
@@ -58,14 +61,104 @@ def parse_single_sentences(raw_sentences):
...
+    #return props_parse(tweet_text)
+
+    # use Spacy to segment tweet to sentences
+    doc = spacy_pipe(unicode(tweet_text))
+    # parse each sentence separately using props
+    props_parses = [props_parse(s.text) for s in doc.sents]
+    # return a merge of the parses
+    return combine_parses(doc, props_parses)
+
+def combine_parses(spacy_doc, props_parses):
```
Comment (OriShapira) [inline src/baseline_automatic_pipeline_system/parse_okr_info.py]: If there is a single sentence, maybe the sentence merging process can be skipped?
This seems like a time-costly procedure that will not be necessary in many tweets.

**Comment by kleinay** [comment]
@OriShapira - thanks for the comments, fixed them. anything else?

**Comment by OriShapira** [comment]
Seems good. Thanks!

**Pull request merged**